enterprise/server/remote_execution/executor/executor.go

```diff
@@ -266,13 +266,19 @@ func (s *Executor) ExecuteTaskAndStreamResults(ctx context.Context, task *repb.E
 	}
 
 	rxInfo := &dirtools.TransferInfo{}
-	// Don't download inputs if the FUSE-based filesystem is enabled.
+	// Don't download inputs or add the CI runner if the FUSE-based filesystem is
+	// enabled.
```

Comment:
I think this breaks fuse support, since in that mode the runner won't exist at all, right? or is fuse able to map it in as is?

It's not a huge deal but we should file a bug / add a TODO to fix.